modules/client-configuration/pages/cobbler.adoc

```diff
@@ -200,162 +176,76 @@ configure-tftpsync.sh proxy1.example.com proxy2.example.com
-== Adding a Distribution to Cobbler
-
-
-If all Cobbler prerequisites have been met and Cobbler is running, you can use the Cobbler server as an installation source for a distribution:
-
-Make installation files such as the kernel image and the initrd image available on the Cobbler server.
-Then add a distribution to Cobbler, using either the Web interface or the command line tools.
-
-For information about creating and configuring {ay} or Kickstart distributions from the {productname} interface, refer to pass:c[xref:FILENAME.adoc#ref.webui.systems.autoinst.distribution[]].
-
-To create a distribution from the command line, use the [command]``cobbler`` command as root:
-
-----
-cobbler distro add --name=`string`--kernel=`path`--initrd=`path`
-----
-
-
-[option]``--name=``[replaceable]``string`` option::
-A label used to differentiate one distribution choice from another (for example, ``sles12server``).
 
-[option]``--kernel=``[replaceable]``path`` option::
-Specifies the path to the kernel image file.
-
-[option]``--initrd=``[replaceable]``path`` option::
-specifies the path to the initial ram disk (initrd) image file.
-
-////
-
-////
-Commented out as per https://bugzilla.suse.com/show_bug.cgi?id=1136611 LKB 2019-05-29
-
-[[advanced.topics.cobbler.addprofile]]
-== Adding a Profile to Cobbler
-
-Once you have added a distribution to Cobbler, you can add profiles.
-
-Cobbler profiles associate a distribution with additional options like {ay} or Kickstart files.
-Profiles are the core unit of provisioning and there must be at least one Cobbler profile for every distribution added.
-For example, two profiles might be created for a Web server and a desktop configuration.
-While both profiles use the same distribution, the profiles are for different installation types.
-
-For information about creating and configuring Kickstart and {ay} profiles in the {productname} interface, refer to pass:c[xref:FILENAME.adoc#ref.webui.systems.autoinst.profiles[]].
-
-Use the [command]``cobbler`` command as root to create profiles from the command line:
+=== Background Information about the Synchronization Process
 
-----
-cobbler profile add --name=string --distro=string [--autoinstall=url] \
-  [--virt-file-size=gigabytes] [--virt-ram=megabytes]
-----
+// Taken from https://bugzilla.suse.com/show_bug.cgi?id=1141603
+// To be edited!!!
 
-[option]``--name=``[replaceable]``string``::
-A unique label for the profile, such as `sles12webserver` or ``sles12workstation``.
+A [command]``cobbler sync`` is a rebuild of every file Cobbler touched.
+On {productname}, [command]``cobbler sync`` does the following actions:
 
-[option]``--distro=``[replaceable]``string``::
-The distribution that will be used for this profile.
-For adding distributions, see pass:c[xref:FILENAME.adoc#advanced.topics.cobbler.adddistro[]].
+1. Run pre-sync triggers.
+This can be any number of shell scripts.
+2. Delete all files and directories that are not allowed in [path]``/var/www/cobbler/``.
```

Comment:
On a SUSE system this directory does not exist. Will cobbler clean a different directory or does this step have no effect? @SchoolGuy ?

Reply:
@mccalmer Yes, you are correct. It cleans the web-directory not specifically `/var/www/cobbler`. Which means @keichwa that you could adjust the path to `/srv/www/cobbler`

Reply:
ok, thanks! fixed locally, I'll push later.